src/azure-cli/azure/cli/command_modules/containerapp/custom.py

```diff
@@ -1621,12 +1620,13 @@ def start_containerappjob_execution_yaml(cmd, name, resource_group_name, file_na
 
 def stop_containerappsjob(cmd, resource_group_name, name, job_execution_name=None, execution_name_list=None):
     try:
+        # todo: remove execution_name_list in future and allow calling with or without job_execution_name
         if execution_name_list is not None:
-            execution_name_list = execution_name_list.split(",")
-            execution_name_list = json.dumps({'jobExecutionName': execution_name_list})
-        r = ContainerAppsJobClient.stop_job(cmd=cmd, resource_group_name=resource_group_name, name=name, job_execution_name=job_execution_name, job_execution_names=execution_name_list)
+            return "--execution-name-list is deprecated. Please use --job-execution-name instead."
```

Comment:
The behavior should keep consistent with before during deprecate time.  You can add log here for customer but should not return this message.

Reply:
What is the behavior if customer input this argument before? Thanks.

Reply:
@Greedygre I see that we were never parsing the list of job execution names correctly passed via --execution-name-list and were getting an error:
![image](https://github.com/user-attachments/assets/9bf0924b-a87d-40d3-b603-6493254b2f33)

Also, in the next RP release we would be removing the JobExecutionList param from /stop API endpoint since it was not defined in the swagger and we want to sync the API with the swagger, hence wanted to remove this option from CLI as well.